lom/src/plugins/plugins_common/plugins_common.go

```diff
@@ -122,8 +122,10 @@ type PluginMetadata struct {
     StartedTime time.Time
```

Comment:
MAX_PLUGIN_RESPONSES_DEFAULT - Note we apply this default only for requests w/o timeout - which are detections. Just wanted to confirm

Reply:
Yes